Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/utils/RomBannerRequestHandler.java

```diff
@@ -0,0 +1,33 @@
+package org.dolphinemu.dolphinemu.utils;
+
+import android.graphics.Bitmap;
+import android.util.*;
+
+import com.squareup.picasso.Picasso;
+import com.squareup.picasso.Request;
+import com.squareup.picasso.RequestHandler;
+
+import org.dolphinemu.dolphinemu.NativeLibrary;
+
+import java.io.IOException;
+import java.nio.IntBuffer;
+
+public class RomBannerRequestHandler extends RequestHandler {
```

Comment:
Could we use a different name than `RomBannerRequestHandler`? We essentially never use the word ROMs in Dolphin to refer to games or disc images (but we do use it for things like the DSP ROMs). What does the rest of the Android code use? "Game"?